src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -55,7 +55,7 @@ protected static function getPhpUnitXmlDir()
 
         // Can't continue
         if (null === $dir) {
-            throw new \RuntimeException('Unable to guess the Kernel directory.');
+            throw new \RuntimeException('Unable to guess the PHPUnit xml directory.');
```

Comment:
I guess this method could be kept for BC and might somehow be useful. So I just update the exceptions messages rather than deprecating it too (same for `getPhpUnitCliConfigArgument`).
But what would be a proper exception message for:

```php
if (!isset($_SERVER['argv']) || false === strpos($_SERVER['argv'][0], 'phpunit')) {
    throw new \RuntimeException('You must override the KernelTestCase::createKernel() method.');
}
```

 some lines above ([L.45](https://github.com/symfony/symfony/pull/22675/files#diff-87a8cc91fd08f09a8a3c0b37d7ddf177R45)) ?